### ISSUE TITLE
error: ‘add_eye_candy_light’ was not declared in this scope when building the map editor

### ISSUE THREAD

**Issue by NetSysFire** [raduprv/Eternal-Lands#179]
I tried to build the latest master (as of writing, b5dbb3e2d75d170fa23a4b6fbf5317ddf9701a6a) and the client builds and works just fine, but the map editor does not.

Running `make` after running `cmake .`:

```
Consolidate compiler generated dependencies of target mapedit.linux.bin
[  1%] Building CXX object CMakeFiles/mapedit.linux.bin.dir
/home/user/eternallands/src/Eternal-Lands-master/eye_candy/eye_candy.cpp.o
/home/user/eternallands/src/Eternal-Lands-master/eye_candy/eye_candy.cpp: In member function ‘void ec::EyeCandy::draw()’:
/home/user/eternallands/src/Eternal-Lands-master/eye_candy/eye_candy.cpp:1991:17: error: ‘reset_eye_candy_lights’ was not declared in this scope
 1991 |                 reset_eye_candy_lights();
      |                 ^~~~~~~~~~~~~~~~~~~~~~
/home/user/eternallands/src/Eternal-Lands-master/eye_candy/eye_candy.cpp:2034:33: error: ‘add_eye_candy_light’ was not declared in this scope
 2034 |                                 add_eye_candy_light(light_pos_world, light_color, light_linear_attenuation);
      |                                 ^~~~~~~~~~~~~~~~~~~
make[2]: *** [CMakeFiles/mapedit.linux.bin.dir/build.make:1462: CMakeFiles/mapedit.linux.bin.dir/home/user/eternallands/src/Eternal-Lands-master/eye_candy/eye_candy.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:83: CMakeFiles/mapedit.linux.bin.dir/all] Error 2
make: *** [Makefile:91: all] Error 2
```

**Comment by gvissers** [comment]
Thanks for reporting, this (as well as an other error) should now be fixed with ecff412.

**Comment by NetSysFire** [comment]
Thanks for the quick fix! I can confirm it builds properly now.
